src/Element.php

```diff
@@ -104,7 +104,7 @@ public function setName($name)
     /**
      * Get value for name
      *
-     * @return string|int
+     * @return mixed
```

Comment:
No. This should only ever be a string or an integer, as the value is used for array keys. Revert, please.

Reply:
@weierophinney i'm use `SplString`